docs/rules/index.md

```diff
@@ -5,42 +5,45 @@ layout: doc
 <!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->
```

Comment:
We also need descriptive text for accessibility purposes, an empty span isn't good enough. 

Also, I think we should keep the plain text versions in the ESLint repo and use the build script to replace "(recommended)" with the icon. That will make it easier to maintain.